src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -1635,8 +1635,7 @@ GetDependentSequencesWithRelation(Oid relationId, List **seqInfoList,
 			attrdefResult = lappend_oid(attrdefResult, deprec->objid);
 			attrdefAttnumResult = lappend_int(attrdefAttnumResult, deprec->refobjsubid);
 		}
-		else if ((deprec->deptype == DEPENDENCY_AUTO || deprec->deptype ==
```

Comment:
hmm, why are we changing this?  Maybe we already find the sequences for identity columns as we have `DEPENDENCY_INTERNAL`? I wonder we needed this just because we have `MarkObjectDistributed` in the other place? I cannot see a reason to change this place, as it seems to find the relevant sequence within the dependency resolution logic